src/context/AppContext.tsx

```diff
@@ -37,14 +39,15 @@ export const AppContextProvider = ({ children }: { children: ReactNode }) => {
   const [user, setUser] = useState(initialUser);
   const [userLoading, setUserLoading] = useState(false);
 
-  const updateUser = useCallback((address) => {
+  const updateUser = useCallback((address?: string) => {
     if (address) {
       try {
         setUserLoading(true);
         const apolloClient = getContext(ContextModule.ApolloClient);
         const query = apolloClient.query({
           query: gql(getCurrentUser),
           variables: { address },
+          fetchPolicy: 'network-only',
```

Comment:
I'm ok with this here, since `updateUser` is conditioned when being called.

But going forward we need to avoid using `network-only` as much as possible, and to try to leverage Apollo's cache.